qmf/src/plugins/messageservices/imap/imapauthenticator.cpp

```diff
@@ -127,7 +127,7 @@ static QByteArray authenticationResponses(QList<QByteArray> &authList, const QMa
                 authPassword = QString::fromLatin1(authList.takeFirst());
                 responseAuthType = QMail::CramMd5Mechanism;
             } else {
-                gResponses[id] = authList;
+                gResponses[id] = authList.takeFirst();
```

Comment:
I changed this a bit since is not need to a map around, there's on instance per account and they don't run at same time.